pkg/client/listers/kubeflow/v1alpha1/tfjob.go

```diff
@@ -34,27 +34,27 @@ type TFJobLister interface {
 	TFJobListerExpansion
 }
 
-// tfJobLister implements the TFJobLister interface.
-type tfJobLister struct {
+// tFJobLister implements the TFJobLister interface.
```

Comment:
Why did you change the capitalization to "tFJobLister" this looks strange to me; I prefer tfJobLister. Since we decided on TFJob (i.e capitalizing TF) i think it makes sense to make both T & F lower case for non public variables.

Reply:
Either way if you do think the capitalization should be changed lets do it a different PR. Since this PR touches so many files it would be good to get it in soon.

Reply:
This is auto-changed by ./hack/update-codegen.sh

Reply:
I think we should not change the code in client/ manually since it will be overrode next time we run the script, and the name is changed to tF since the job is renamed from TfJob to TFJob. https://github.com/tensorflow/k8s/pull/332

Reply:
You're right. I didn't realize this was autogenerated.